packages/cli/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "@types/debug": "^4.1.10",
-    "oclif": "^4.0.3",
+    "oclif": "^4.13.8",
```

Comment:
We don't need to bump a dev dep just to clear the advisory, the package-lock should suffice.